packages/react-composites/src/composites/CallComposite/ConfigurationScreen.tsx

```diff
@@ -13,54 +13,70 @@ import { getCallingSelector } from '@internal/calling-component-bindings';
 import { Stack } from '@fluentui/react';
 import { LocalPreview } from './LocalPreview';
 import {
-  configurationStackTokens,
+  configurationStackTokensDesktop,
+  configurationStackTokensMobile,
   configurationContainer,
   selectionContainerStyle,
-  titleContainerStyle
+  startCallButtonContainerStyleDesktop,
+  startCallButtonContainerStyleMobile,
+  startCallButtonStyleMobile,
+  titleContainerStyleDesktop,
+  titleContainerStyleMobile
 } from './styles/CallConfiguration.styles';
 import { useLocale } from '../localization';
 
 /**
  * @private
  */
 export interface ConfigurationScreenProps {
+  mobileView?: boolean;
   startCallHandler(): void;
 }
 
 /**
  * @private
  */
 export const ConfigurationScreen = (props: ConfigurationScreenProps): JSX.Element => {
-  const { startCallHandler } = props;
+  const { startCallHandler, mobileView = false } = props;
 
   const options = useAdaptedSelector(getCallingSelector(OptionsButton));
   const localDeviceSettingsHandlers = useHandlers(LocalDeviceSettings);
   const { video: cameraPermissionGranted, audio: microphonePermissionGranted } = useSelector(devicePermissionSelector);
 
   const locale = useLocale();
-  const title = locale.strings.call.configurationPageTitle;
+  const title = (
+    <Stack.Item className={mobileView ? titleContainerStyleMobile : titleContainerStyleDesktop}>
+      {locale.strings.call.configurationPageTitle}
+    </Stack.Item>
+  );
 
   return (
     <Stack
-      horizontal
-      wrap
-      horizontalAlign="center"
+      horizontal={!mobileView}
+      horizontalAlign={mobileView ? 'stretch' : 'center'}
       verticalAlign="center"
-      tokens={configurationStackTokens}
+      tokens={mobileView ? configurationStackTokensMobile : configurationStackTokensDesktop}
       className={configurationContainer}
```

Comment:
You will need to change the padding and min-width in configurationContainer as they depend on tokens value (compensation of the inner stack due to wrapping)

Reply:
Updated 👍